ppml/trusted-python-toolkit/Dockerfile

```diff
@@ -2,29 +2,45 @@ ARG BASE_IMAGE_NAME
 ARG BASE_IMAGE_TAG
 ARG BIGDL_VERSION=2.2.0-SNAPSHOT
 ARG TINI_VERSION=v0.18.0
+ARG JDK_VERSION=11
 
 #Stage.1 Flask & Numpy & Pandas
 FROM $BASE_IMAGE_NAME:$BASE_IMAGE_TAG
 ARG http_proxy
 ARG https_proxy
+ARG JDK_VERSION
 ARG TINI_VERSION
+ENV JAVA_HOME				/opt/jdk${JDK_VERSION}
 ENV TINI_VERSION                        $TINI_VERSION
 
 ADD ./examples/ /ppml/examples
+ADD ./frontend.jar /ppml/torchserve
```

Comment:
better to fork the repo and build this jar

Reply:
Finished.